app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -27,14 +27,9 @@ class MainActivity : AppCompatActivity() {
             supportFragmentManager.beginTransaction()
                     .replace(R.id.container, MainFragment.newInstance())
                     .commitNow()
-        }
 
-       /* NOTE: Here I was trying to display the weather data under the actual temp value, but
-          the way the MainFragment is setup with the ViewModel and  MainActivity makes this a bit more confusing.
-          It throws a null pointer reference error and the app will crash */
```

Comment:
I feel like any comment that is not helping the program shouldn't be in the program.

Reply:
Good idea to remove.

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.